test-data/src/main/java/dev/futa/tutorial/pesel/PeselDataSetSupplier.java

```diff
@@ -0,0 +1,41 @@
+package dev.futa.tutorial.pesel;
+
+import com.google.common.base.Preconditions;
+
+import java.time.LocalDate;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static dev.futa.tutorial.pesel.Gender.FEMALE;
+import static dev.futa.tutorial.pesel.Gender.MALE;
+
+public enum PeselDataSetSupplier {
+  INSTANCE;
+
+  public Set<DecodedPeselData> getValidPeselSet() {
+
+    Preconditions.checkArgument(true, "asd");
+
+    // todo extend for other examples
+    return Stream.of(
```

Comment:
Zamiast korzystać z Stream.of, a potem ładować to do set'a przez collector (który i tak po pozbieraniu wszystkiego do HashSet korzysta z Set.of()), można od razu skorzystać z Set.of()

Reply:
true. done.